### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [mesubhash/TeaCozy#4]
https://github.com/mesubhash/TeaCozy/blob/79d978f811e6bc4e8f74f2d2ea2fdf0636320c84/index.html#L26

All images should have descriptive alt text.  This helps the visually impaired through the use of their screen readers.  Also, this allows your site to pass HTML5 validation.
